ui/src/testing/stubs/workspace-admin-api-stub.ts

```diff
@@ -1,12 +1,9 @@
 import { WorkspaceAdminApi } from 'generated/fetch';
 
-import { stubNotImplementedError } from './stub-utils';
 import { workspaceStubs } from './workspaces';
 
 export class WorkspaceAdminApiStub extends WorkspaceAdminApi {
   constructor(public workspaces = workspaceStubs) {
-    super(undefined, undefined, (..._: any[]) => {
-      throw stubNotImplementedError;
-    });
+    super(undefined);
```

Comment:
This ultimately extends BaseApi which only takes one parameter: a configuration object.